src/lib/schemas/service.js

```diff
@@ -249,7 +249,7 @@ const schema = {
     rules: [
       v.isString(),
       (name, value, data) => ({
-        valid: !data.hasFee || value.length,
+        valid: data.feeCondition === "gratuit",
```

Comment:
marche pas, ce n'est valide que si c'est gratuit, alors que tu veux gratuit ou non vide

```suggestion
        valid: data.feeCondition === "gratuit" || value.length,
```

Reply:
En effet... en effet... :/